torchvision/transforms.py

```diff
@@ -52,33 +52,38 @@ def __call__(self, pic):
         """
         if isinstance(pic, np.ndarray):
             # handle numpy array
-            img = torch.from_numpy(pic.transpose((2, 0, 1)))
+            if pic.ndim == 2:
+		pic = pic[np.newaxis]
+	    elif pic.ndim == 3:
+		pic.transpose((2, 0, 1))
+	    else:
+		raise ValueError('only 2D and 3D images accepted, got {}D image'.format(pic.ndim)
             # backward compatibility
-            return img.float().div(255)
+            return torch.from_numpy(pic).float().div(255.)
 
         if accimage is not None and isinstance(pic, accimage.Image):
-            nppic = np.zeros([pic.channels, pic.height, pic.width], dtype=np.float32)
+            nppic=np.zeros([pic.channels, pic.height, pic.width], dtype=np.float32)
```

Comment:
PEP recommendations are to surround assignment operators with a space (except for in kwargs) - not sure why the linter didn't pick this up but this is how we've done it across the codebase, so I'd rather not change it in this file.